index.html

```diff
@@ -174,5 +174,26 @@ <h2>Skills</h2>
   </div>
   <div class="line"></div>
   </aside>
+  <footer>
+    <div class="footer-heading">
+    <h2 class="footer-heading">Contact me</h2>
+    <p id="description-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  </div>
+  <div class="decription">
+    <form action="https://formspree.io/f/mbjwpoar">
+      <label>
+        <input type="text" name="user" value="Yeremias" maxlength="30" required>
+      </label>
+      <label>
+        <input type="email" name="email" value="natajayanj14@gmail.com" required>
+      </label>
```

Comment:
- [ ] You are almost there! It would be better to use `placeholder` instead of `value` this way the fields will not have a default value and this will avoid sending wrong data.

Reply:
Hello, in the Figma design it has a dark color than placeholder that is why I put value should I change it or not?

Reply:
@smunozmo I asked a question in above answer it please. Thanks a lot.